test/e2e/tests/releases/utils/methods.ts

```diff
@@ -59,6 +52,9 @@ export const archiveRelease = async ({
         ],
       },
     })
+  } catch (error) {
+    console.error('Error archiving release:', error)
+    throw error
```

Comment:
Not sure why we need to both output and throw – throwing will surface the error anyway, so this seems redundant?

Reply:
That's true, I was thinking about doing just the console log but then remembered about the throw and forgot to remove the console log. I'll just leave the throw.

Reply:
then you can just remove the try catch altogether :)

Reply:
haha was just about to comment that 😂